### ISSUE TITLE
Open inside the app: democracy-app.de/antrag/* 

### ISSUE THREAD

**Issue by gloschtla** [demokratie-live/democracy-client#784]
## 🚀 Feature
When I click on the link to "democracy-app.de/antrag/*" then my smartphone should offer to open this link inside the DEMOCRACY app. I would always like to open it within the app, instead of within a browser.

**Comment by ulfgebhardt** [comment]
Related: https://github.com/demokratie-live/democracy-client/issues/585
Works for iOS but not for (all) Android

**Comment by visionmakery** [comment]
<img width="770" alt="Bildschirmfoto 2019-05-07 um 12 18 48" src="https://user-images.githubusercontent.com/32302889/57292598-5b587180-70c2-11e9-9f52-79294db9f643.png">

**Comment by gloschtla** [comment]
@Frank
Nein, das funktioniert nicht. Ich habe es soeben erneut gestestet. Und zwar in der letzten Version 1.0.5, die bei [F-Droid](https://f-droid.org/en/packages/de.democracydeutschland.app/) verfügbar ist; installiert in Android 7.1.2
